projects/vdk-plugins/vdk-meta-jobs/tests/test_meta_job.py

```diff
@@ -305,6 +305,111 @@ def test_meta_job_circular_dependency(httpserver: PluginHTTPServer):
         assert len(httpserver.log) == 0
 
 
+def test_meta_job_wrong_job_type(httpserver: PluginHTTPServer):
+    jobs = [("job" + str(i), [200], "succeeded") for i in range(1, 5)]
+    api_url = _prepare(httpserver, jobs)
+
+    with mock.patch.dict(
+        os.environ,
+        {"VDK_CONTROL_SERVICE_REST_API_URL": api_url},
+    ):
```

Comment:
You can avoid repeating this chunk so many times is you wrap these tests in a class and including it in a setUp method; read more here https://docs.python.org/3/library/unittest.html#unittest.TestCase.setUp